server/requirements.txt

```diff
@@ -6,4 +6,4 @@ wooper==0.4.2
 pymongo==2.8
 Eve==0.6.0
 
--e git+git://github.com/superdesk/superdesk-core@8e843fc#egg=Superdesk-Core==0.0.1-dev
\ No newline at end of file
+-e git+git://github.com/superdesk/superdesk-core@c280342#egg=Superdesk-Core==0.0.1-dev
```

Comment:
PR https://github.com/superdesk/superdesk/pull/1382 is not merged yet. On PR merge it will be a conflict and a rebase will be needed on the last merged one.